Core/StitchCoreSDK/Sources/StitchCoreSDK/Auth/Internal/CoreStitchAuth+LoginAndLink.swift

```diff
@@ -0,0 +1,269 @@
+import Foundation
+import MongoSwift
+
+/**
+ * Extension functions for `CoreStitchAuth` to perform login
+ */
+extension CoreStitchAuth {
+    /**
+     * Authenticates the `CoreStitchAuth` using the provided `StitchCredential. Blocks the current thread until the
+     * request is completed.
+     */
+    public func loginWithCredentialInternal(withCredential credential: StitchCredential) throws -> TStitchUser {
```

Comment:
[style] These methods should be named to correspond to the parameter, e.g., `loginInternal(withCredential:)`